evap/evaluation/management/commands/import_cms_data.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import urllib.parse
+
+import requests
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from evap.evaluation.management.commands.tools import log_exceptions
+from evap.evaluation.models import Semester
+from evap.staff.importers.json import JSONImporter
+
+logger = logging.getLogger(__name__)
+
+
+@log_exceptions
+class Command(BaseCommand):
+    help = "Downloads the JSON file with the CMS data for a given semester and imports it."
+
+    def handle(self, *args, **options):
+        logger.info("import_cms_data called.")
+
+        for semester in Semester.objects.filter(default_course_end_date__isnull=False).exclude(cms_name=""):
+            logger.info("Processing %s.", semester.name_en)
+
+            semester_string = urllib.parse.quote(semester.cms_name)
+            url = settings.CMS_DATA_DOWNLOAD_URL.format(semester_string)
+            filename = "CMS.json"
+            download_successful = download(url, filename)
+            if not download_successful:
+                continue
+
+            import_data(semester, filename)
```

Comment:
Do we need the custom chunking logic? Otherwise we can greatly simplify this by using the builtin .content / .text. We also don't need to write to a file, since we load all of it back into memory later anyways:
```suggestion
            for _ in range(RETRIES):
                try:
                    json_contents = requests.get(url, timeout=TIMEOUT).text
                    break
                except requests.exceptions.Timeout:
                    logger.warning("Download timed out: %s", url)
            else:
                logger.warning("Giving up.")
                continue

            JSONImporter(semester, semester.default_course_end_date).import_json(json_contents)
```